Package.swift

```diff
@@ -0,0 +1,35 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "CocoaLumberjack",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "CocoaLumberjack",
+            type: .dynamic,
+            targets: ["CocoaLumberjack"]),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(name: "CocoaLumberjack",
```

Comment:
Would there be a second target for the swift parts?

Reply:
Well, I am not sure how Swift Package manager solves dependencies for mixed targets. Moreover, I suppose that it doesn't support pure Objective-C targets, right?

Reply:
I don't think it supports mixed (Objective-C/Swift) targets right now. So having an Objective-C and a swift target would make sense.

Reply:
@diederich It doesn't. I filtered out swift files.

Reply:
The Xcode project and the cocoa pods file seem to have 2 targets. One is "core", the other "CocoaLumberjackSwift" 
My guess would have been that Package.swift would also export those 2 options (and thus have those 2 targets)
So maybe add another target, which depends on this first one and includes the swift files? (I guess this could either be handled by excluding non swift files, or creating a separate directory for swift parts)

Reply:
@diederich 
Consider latest commit. 
I comment out all stuff related to additional target in case of "unknown symbol DDLogLevel and its friends".